code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -276,7 +281,7 @@
 		owner.visible_message(span_warning("[owner] commands their army to action, mutating them into rats!"))
 
 	else if(LAZYLEN(SSmobs.cheeserats) < cap)
-		new /mob/living/basic/mouse(owner.loc, owner.faction)
+		new /mob/living/basic/mouse(owner.loc)
```

Comment:
Rats do use faction checks, wouldn't it be better to add the proper support to `/mob/living/basic/mouse/Initialize(mapload)` instead?